src/GraphQL/Execution/ExecutionStrategy.cs

```diff
@@ -93,10 +95,7 @@ public static void SetSubFieldNodes(ExecutionContext context, ObjectExecutionNod
                 var name = kvp.Key;
                 var field = kvp.Value;
 
-                if (!ShouldIncludeNode(context, field.Directives))
```

Comment:
@Shane32 Note that this line was not necessary because fields has already been collected and filtered in `Fields.CollectFields`. See `ShouldIncludeNode` usages.